ci-operator/step-registry/upi/gcp/ssh-bastion/pre/upi-gcp-ssh-bastion-pre-commands.sh

```diff
@@ -0,0 +1,278 @@
+#!/bin/bash
```

Comment:
Per the following code, seem like this script is not only create a ssh bastion, also a proxy bastion host, so maybe we need to change the script name to `ci-operator/step-registry/upi/gcp/bastion/upi-gcp-bastion-commands.sh`.